stock_picking_invoice_product_group/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    related_picking_ids = fields.Many2many(
```

Comment:
Yes, migration script is for upgrades (you only need to bump version number). If any other task has to be made on first install, then you need to put a post_init_hook. See the migration scripts we have done for handling l10n_es evolution: https://github.com/OCA/l10n-spain/tree/8.0/l10n_es/migrations